src/test/common.go

```diff
@@ -0,0 +1,15 @@
+package test
+
+import (
+	"flag"
+	"io/ioutil"
+	"log"
+)
+
+var isLogOn = flag.Bool("log", false, "Log output to screen")
+
+func checkLogOutput() {
+	if !*isLogOn {
+		log.SetOutput(ioutil.Discard)
+	}
+}
```

Comment:
common.go was to be created, because otherwise isLogOn would be declared more than once